.github/workflows/collect-webkit.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, collect data for states that use webkit, and run tests on the generated files
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Collect data for webkit states
+
+on:
+  schedule:
+    - cron: '0 22 * * 0'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code repo
+      uses: actions/checkout@v2
+      with:
+        path: 'code_repo'
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Create venv and install dependencies
+      run: |
+        pwd
+        cd code_repo
+        make create-install
+    - name: Remove old data files
+      run: rm code_repo/public/*.json
+    - name: Collect webkit states
+      env:
+        QT_QPA_PLATFORM: "minimal"
+        QT_DEBUG_PLUGINS: 1
+      run: |
+        pwd
+        cd code_repo
+        source venv/bin/activate && inv collect webkit
+    - name: Test new data files
+      run: |
+        pwd
+        cd code_repo
+        ls -ls public
+        make test
+    - name: Checkout data repo
+      uses: actions/checkout@v2
+      with:
+        path: 'data_repo'
+        ref: 'collect-data'
+    - name: Move files to data repo
+      run: |
+        pwd
+        cd data_repo
+        ls -ls public
+        mv ../code_repo/public/*.json public -f
+    - name: Commit files
+      run: |
+        pwd
+        cd data_repo
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git add ./public
+        git commit -m "Collect new data files"
+        git push origin collect-data
```

Comment:
Can we add a tag with the date?  We want a stable reference as well as a reference to 'latest' (this branch).